File: cEs.Portal/wwwroot/js/ces/Solicitacao.js
function Salvar() {
    var url = "/Solicitacao/Salvar";
    alert("Oi");
    var data = $('form').serialize();
    var valida = true;
    if ($('#Nome').val() == undefined || $('#Nome').val() == "" || $('#Nome').val() == 'null' || $('#Nome').val() == null ||
        $('#Endereco').val() == undefined || $('#Endereco').val() == "" || $('#Endereco').val() == 'null' || $('#Endereco').val() == null ||
        $('#Numero').val() == undefined || $('#Numero').val() == "" || $('#Numero').val() == 'null' || $('#Numero').val() == null ||
        $('#Bairro').val() == undefined || $('#Bairro').val() == "" || $('#Bairro').val() == 'null' || $('#Bairro').val() == null ||
        $('#Municipio').val() == undefined || $('#Municipio').val() == "" || $('#Municipio').val() == 'null' || $('#Municipio').val() == null ||
        $('#UF').val() == undefined || $('#UF').val() == "" || $('#UF').val() == 'null' || $('#UF').val() == null ||
        $('#Cep').val() == undefined || $('#Cep').val() == "" || $('#Cep').val() == 'null' || $('#Cep').val() == null ||
        $('#Contato').val() == undefined || $('#Contato').val() == "" || $('#Contato').val() == 'null' || $('#Contato').val() == null ||
        $('#Celular').val() == undefined || $('#Celuler').val() == "" || $('#Celular').val() == 'null' || $('#Celular').val() == null ||
        $('#Telefone').val() == undefined || $('#Telefone').val() == "" || $('#Telefone').val() == 'null' || $('#Telefone').val() == null ||
        $('#Email').val() == undefined || $('#Email').val() == "" || $('#Email').val() == 'null' || $('#Email').val() == null ||
        $('#SolicitacaoStatusId').val() == undefined || $('#SolicitacaoStatusId').val() == "" || $('#SolicitacaoStatusId').val() == 'null' || $('#SolicitacaoStatusId').val() == null ||
        $('#Assunto').val() == undefined || $('#Assunto').val() == "" || $('#Assunto').val() == 'null' || $('#Assunto').val() == null)
    {
        valida = false;
    };
    if (valida) {
        $.ajax({
            url: url
            , data: data
            , type: "POST"
            , datatype: "html"
            , cache: false
            , async: true
            , success: function (data) {
                alert("E-Mail enviado com sucesso");
                window.location.href = window.location.protocol + '//' + window.location.hostname + ':' + window.location.port + '/Solicitacao/Index';
            }
        })
    }
}

File: Domain/cEs.Domain/Entities/Comercial/Solicitacao.cs
using cEs.Domain.Interface.Entities.Comercial;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Text;

namespace cEs.Domain.Entities.Comercial
{
    [DataContract]
    public class Solicitacao : ISolicitacao
    {
        [DataMember]
        public long? SolicitacaoId { get; set; }
        [DataMember]
        public string Nome { get; set; }
        [DataMember]
        public string Endereco { get; set; }
        [DataMember]
        public int? Numero { get; set; }
        [DataMember]
        public string Complemento { get; set; }
        [DataMember]
        public string Bairro { get; set; }
        [DataMember]
        public string Municipio { get; set; }
        [DataMember]
        public string UF { get; set; }
        [DataMember]
        public string Cep { get; set; }
        [DataMember]
        public string Contato { get; set; }
        [DataMember]
        public string Celular { get; set; }
        [DataMember]
        public string Telefone { get; set; }
        [DataMember]
        public string Email { get; set; }
        [DataMember]
        public string Assunto { get; set; }
        [DataMember]
        public string Procurar { get; set; }
        [DataMember]
        public string SolicitacaoStatusId { get; set; }
        [DataMember]
        public DateTime? Data { get; set; }

        [DataMember]
        public string Status { get; set; }
      }
   
}
